articles/how-to-use-kanban-to-manage-your-next-project/index.md

```diff
@@ -0,0 +1,132 @@
+- "to do", "doing" and "done"
+- "waiting", "in progress" and "completed"
+- "requested", "in progress" and "done"
+
+The diagram below shows a sample simple Kanban board with three columns. 
+
+![Simple Kanban board](/engineering-education/how-to-use-kanban-to-manage-your-next-project/simple-kanban-board.jpg)
+
+[Image source](https://hygger.io/use-cases/kanban-board/)
+
+Kanban boards can be made on walls, whiteboards, or with [digital tools](#Kanban-Software-Tools). This can be divided into two categories:
+
+- Physical Kanban boards: Created on physical walls or whiteboards.
+- Digital Kanban boards: created with software tools and are in digital form
+
+The diagram below shows the main components of a Kanban board.
+
+![Kanban board components](/engineering-education/how-to-use-kanban-to-manage-your-next-project/kanban-board-components.jpg)
+
+[Image source](https://hygger.io/use-cases/kanban-board/)
+
+#### Kanban cards
+
+Work is divided into smaller tasks. Each task is represented by a card called a Kanban card. A Kanban card has a title, description, and owner. Other details that can be included in the Kanban card are the due date and card history. Card history shows the movement of a card from one stage to another. Cards on digital Kanban boards can have more details than those on physical Kanban board. The size of the task on a card is crucial. A task should not be too small or too big. For example, it should not take weeks to work on one card. It should also be big enough in such a way that you don't have to move or change a card every few hours.
+
+#### Work in progress limit (WIP limit)
+
+Work in progress limit is a value (usually put below the Kanban board columns) that limit the number of work items under a step. WIP limits are used to expose bottlenecks in the workflow and maximize workflow. WIP limit is set based on team capacity. WIP limit reduces the wait time from previous steps and reduces stress on resources.
+
+#### Commitment point and delivery point
+
+The commitment point is when team members pull a task from the backlog or the customer requests and commit to working on it. The delivery point is the end of Kanban workflow, and the product is delivered to the customer. The duration between the commitment point and the delivery point is called lead time.
+
+#### Swimlanes
+
+Swin lanes are horizontal lines on a Kanban board. They are used to visually separate different tasks on the same Kanban board. They help organize similar tasks together. For example, in software development, tasks that involve fixing some bugs can be put in the same swim lane.
+
+### Benefits of Kanban
+
+There are numerous advantages of using Kanban to manage your project, including:
+- Planning flexibility: With no fixed roles and processes, Kanban is flexible to adopt market changes.
```

Comment:
> Planning flexibility: With no fixed roles and processes, Kanban is flexible to adopt market changes.

It would be great if you could make the sub-headings as bold.

For example,

**Planning flexibility:** With no fixed roles and processes, Kanban is flexible to adopt market changes.